package-namespace-migration.md

```diff
@@ -1,6 +1,6 @@
-# Node.js Package Namespace Migration Plan
+# Node.js Package Scope Migration Plan
 
-Packages intended to be used directly by general developers should be published under the `@nodejs` namespace. Packages intended to be used only by Node.js contributors should be published under the `@node-core` namespace.
+Packages intended to be used directly by general developers should be published under the `@nodejs` scope. Packages intended to be used only by Node.js contributors should be published under the `@node-core` scope.
```

Comment:
not a big deal, but why not nodejs-core?

Reply:
See discussion here: https://github.com/nodejs/TSC/issues/1178

Adding this to PR description

Reply:
Gotcha. I don’t see any reasons there to avoid the consistency, but it’s not a big deal either way.